apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/derived/DerivedDbTable.java

```diff
@@ -77,6 +77,9 @@ protected DerivedDbTable(String table) {
     @Override
     public void trim(int height) {}
 
+    @Override
+    public void trim(int height, boolean isSharding) {}
```

Comment:
I think this is a good place to implement the default behavior of the new trim method, instead of explicit implementation in each 'derived table' class.

Reply:
Done